falcon/routing/converters.py

```diff
@@ -132,8 +152,22 @@ def convert(self, value):
             return None
 
 
+class PathConverter(BaseConverter):
+    """Field converted used to match the rest of the path.
+
+    This field converter matches the all the remaining url path,
+    returning it as a string.
```

Comment:
We should also note what the reader should expect when the path converter comes at the end of a path that is already consumed. For example, given:

Template: `'/foo/bar/{foo:path}'`
Path Requested: `'/foo/bar'`

The path will not match. Also describe what happens when there's a trailing forward-slash in the requested path?

`'/foo/bar/'`